### PR TITLE
docs: document the Opsgenie "teams" responder type behavior

### PR DESCRIPTION
**Pull request by multani**
It has been changed in #2685 but the behavior is not really documented:

* `teams` supports a comma-separated list of teams, through the `name` fields only
* if the list of teams is empty, no team responders are sent to Opsgenie

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -654,6 +654,10 @@ responders:
 type: <tmpl_string>
 ```
 
+The `teams` responder is configured using the `name` field. This field can
```
Comment (simonpasquier) [inline docs/configuration.md:657]: can you move this after L653 so it's more visible to users?

Comment (multani) [reply docs/configuration.md:657]: @simonpasquier as a comment inside the "yaml" block then?

**Review by simonpasquier — APPROVED** [review]
lgtm

File: docs/configuration.md (hunk 1 above)
Comment (simonpasquier) [reply docs/configuration.md]: yes perfect!

**Comment by multani** [comment]
@simonpasquier Should I amend my last commit to add the signature and fix the DCO workflow?

**Comment by simonpasquier** [comment]
@multani yes please.
The CI failures are suspect and unrelated to your change though.

**Comment by multani** [comment]
> @multani yes please.
> The CI failures are suspect and unrelated to your change though.

I amended my last commit to include the signature, the DCO check passes now.

**Comment by simonpasquier** [comment]
I meant the `ci/circleci:test` failures :)

**Comment by multani** [comment]
> I meant the `ci/circleci:test` failures :)

Yeah, I figured this out ;) (it seems it doesn't work on `main` as well)

**Comment by multani** [comment]
@simonpasquier I just remembered I have this PR open, could we merge it perhaps? :D

**Pull request merged**